### DISCUSSION TITLE
Вариации

### DISCUSSION THREAD

**Discussion by Mekhet** [wpcraft-ru/wooms#447] [Q&A]
Здравствуйте! Что если атрибуты на сайте не через стандартные функции? У нас тема Avada  и в ней тип атрибутов товара можно выбрать свой, например отображение цвета в виде кнопки, или просто кнопки, в таком случае синхронизация вариаций происходить не будет?

**Comment by aiiddqd** [comment]
сложный вопрос. нужна диагностика.
можете сделать сами и найти ответ.
можно тут https://wpcraft.ru/catalog/diagnosis/
